src/AfriCC/EPP/ContactTrait.php

```diff
@@ -261,6 +261,19 @@ public function appendIdentity($path, $identity)
         }
     }
 
+    private function appendIsFinish($path, $finish)
```

Comment:
all these changes should most prolly be made as an extension rather than hacking into the core directly